src/System.Text.Encodings.Web/src/System/Text/Encodings/Web/CodePointFilter.cs

```diff
@@ -49,46 +50,46 @@ public CodePointFilter(params UnicodeRange[] allowedRanges)
         }
 
         /// <summary>
-        /// Allows the character specified by <paramref name="c"/> through the filter.
+        /// Allows the character specified by <paramref name="character"/> through the filter.
         /// </summary>
         /// <returns>
         /// The 'this' instance.
         /// </returns>
-        public CodePointFilter AllowChar(char c)
+        public CodePointFilter AllowChar(char character)
         {
-            _allowedCharsBitmap.AllowCharacter(c);
+            _allowedCharsBitmap.AllowCharacter(character);
             return this;
         }
 
         /// <summary>
-        /// Allows all characters specified by <paramref name="chars"/> through the filter.
+        /// Allows all characters specified by <paramref name="characters"/> through the filter.
         /// </summary>
         /// <returns>
         /// The 'this' instance.
         /// </returns>
-        public CodePointFilter AllowChars(params char[] chars)
+        public CodePointFilter AllowChars(params char[] characters)
         {
-            if (chars != null)
+            if (characters != null)
```

Comment:
No problem with the cleanup happening in a separate PR, but I feel it does have to happen before shipping. Changing it later (depending on the change) would be a breaking change. And... we all know these things often get "forgotten" or punted if they're left too late. Totally your call.